abel/basex.py

```diff
@@ -74,7 +76,8 @@ def _abel_basex_core(data, center='auto', n='auto',
                     the center column of the image
                 * tuple (x,y) -
                     the center of the image in (x,y) format
-                * If center='auto', it is set to (data.shape[0]//2, data.shape[1]//2)
+                * If center='auto', it is set to (data.shape[1]//2, data.shape[0]//2)
+                * if n='com', it is set to the center of mass of data
```

Comment:
Typo here, `* if n="com" it is set...` should be `* if center="com" it is set...`